rpc/lib/client/ws_client.go

```diff
@@ -254,11 +254,13 @@ func (c *WSClient) reconnect() error {
 		c.mtx.Unlock()
 	}()
 
+	_1sAsNs := float64(time.Second.Nanoseconds())
 	for {
-		c.Logger.Info("reconnecting", "attempt", attempt+1)
+		jitter := time.Duration(rand.Float64() * _1sAsNs)
```

Comment:
how big can jitter be? 1sAsNs is 1000000000, right? so given rand.Float() returned 1.0, max jitter is 1000000000 seconds?

Reply:
maybe `oneSecondAsNanoseconds`... :0?

Reply:
yeah, don't like the name too

Reply:
`billionNS` ?

Reply:
`how big can jitter be? 1sAsNs is 1000000000, right? so given rand.Float() returned 1.0, max jitter is 1000000000 seconds?`
So time.Durations units are in ns NOT seconds https://golang.org/pkg/time/#Duration, therefore the max value of `rand.Float() * 1sAsNs is 1e9ns * 1.0 --> 1s` 

Ahaha we've bikeshed on the name. @silasdavis that seems too verbose to me :), I can try to fly with @ebuchman's but I believe I was trying to convey the number of nano seconds within a second without having to whip out a calculator to convert between seconds and nanoseconds.